robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -188,6 +204,76 @@ function initRosWeb () {
 }
 
 /* functions used in main code */
+
+function rosTimestamp () {
+  // next few lines taken and adjusted from roslibjs action server example
+  let currentTime = new Date()
+  let secs = currentTime.getTime()/1000 // seconds before truncating the decimal
+  let secsFloored = Math.floor(secs) // seconds after truncating
+  let nsecs = Math.round(1000000000*(secs-secsFloored)) // nanoseconds since the previous second
```

Comment:
I say rename this `nanoSecs` to make the name clearer

Reply:
done